### PR TITLE
chore(#8958): Translation to Spanish L1200 to last

### PR DESCRIPTION
**Pull request by latin-panda**
# Description
#8958

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:last-spanish-translations/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:last-spanish-translations/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:last-spanish-translations/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by latin-panda** [comment]
@lorerod, the last round of translations! Please review, thank you

File: api/resources/translations/messages-es.properties [hunk 1]
```diff
@@ -1197,221 +1197,237 @@ report.pregnancy_visit.referral_follow_up_needed = Referencia de Seguimiento Nec
...
+report.subject.unknown = Desconocido
 report.updated = Informe actualizado
-reported_date = Fecha de envío
-reporter.link =
-reporter.section.help = 
-reporter.section.title =
-reporting_unit_not_found = No encontramos a su centro de salud. Por favor corrijalo y reenvie el reporte.
+reported_date = Fecha del informe
+reporter.link = Ir al Medic Reporter
+reporter.section.help = Usar Medic Reporter para probar los formularios entrantes de SMS
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:1211]:
```suggestion
reporter.section.help = Usar Medic Reporter para probar los formularios de SMS entrantes.
```

File: api/resources/translations/messages-es.properties [hunk 2]
```diff
@@ -1197,221 +1197,237 @@ report.pregnancy_visit.referral_follow_up_needed = Referencia de Seguimiento Nec
...
+reporter.section.title = Medic Reporter
+reporting_unit_not_found = No encontramos a el centro de salud. Por favor corrijalo y reenvíe el reporte.
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:1213]:
```suggestion
reporting_unit_not_found = No encontramos el centro de salud. Por favor corrijalo y reenvíe el reporte.
```

File: api/resources/translations/messages-es.properties [hunk 3]
```diff
@@ -1197,221 +1197,237 @@ report.pregnancy_visit.referral_follow_up_needed = Referencia de Seguimiento Nec
...
-sys.incorrect_type =
+sys.empty = El mensaje está en blanco.
+sys.facility_not_found = No se encontró el centro de salud.
+sys.form_not_found = Formulario '{{form}}' no encontrado.
+sys.incorrect_type = Tipo de campo incorrecto {{key}}, esperado {{expectedType}}.
 sys.missing_fields = Campo invalido o faltante\: {{fields}}.
-sys.recipient_not_found = No se encontro destinatario para el mensaje.
-targets.active_pnc.title =
+sys.recipient_not_found = No se encontró el destinatario para el mensaje.
+targets.active_pnc.title = Mujeres en periodo activo de PNC
```
Comment (lorerod) [inline api/resources/translations/messages-es.properties:1302]: I'm thinking about this just now.  Perinatal Care, or "PNC" in Spanish, is called "Cuidado Perinatal" or "CP." Do you think we should use the Spanish abbreviation?

**Review by lorerod — CHANGES_REQUESTED** [review]

File: api/resources/translations/messages-es.properties (hunk 3 above)
Comment (latin-panda) [reply api/resources/translations/messages-es.properties:1302]: Yes! I made the change in all occurrences

**Review by lorerod — APPROVED** [review]

**Pull request merged**